sonar-ws/src/main/protobuf/ws-components.proto

```diff
@@ -75,16 +75,16 @@ message SearchProjectsWsResponse {
 }
 
 message Component {
-  optional string organization = 1;
-  optional string id = 2;
-  optional string key = 3;
-  optional string refId = 4;
-  optional string refKey = 5;
-  optional string projectId = 6;
-  optional string name = 7;
-  optional string description = 8;
-  optional string qualifier = 9;
-  optional string path = 10;
-  optional string language = 11;
-  optional bool isFavorite = 12;
+  optional string organization = 12;
+  optional string id = 1;
+  optional string key = 2;
+  optional string refId = 3;
+  optional string refKey = 4;
+  optional string projectId = 5;
+  optional string name = 6;
```

Comment:
wrong numbers, they should not be changed in order to keep backward-compatibility.